internal/services/springcloud/spring_cloud_java_deployment_resource.go

```diff
@@ -324,6 +330,12 @@ func resourceSprintCloudJavaDeploymentSchema() map[string]*pluginsdk.Schema {
 			ValidateFunc: validate.SpringCloudAppID,
 		},
 
+		"active": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Computed: true,
+		},
```

Comment:
this is missing `addon_json`?

Reply:
`addon` is only supported in `enterprise` sku, and `enterprise` sku doesn't support java deployment, so no need to support `addon` here.